### DISCUSSION TITLE
Trouble choosing correct premium package

### DISCUSSION THREAD

**Discussion by Airbust72** [opengisch/QField#4085] [Q&A]
Hi, 

I have a team with 2 Head of project who will be in charge of updating our different maps and pushing them into the cloud. 
We will then have a team of 10+ workers connect to the cloud as readers to get the information they need. 

Do we need to have 1 Organization account per person ? Or 2 for the head of projects and the workers can access with free accounts ? 

Thank you for your time,

**Comment by ZsanettMed** [comment]
Hi, 
You need an organization account with 12+ members (EarlyBee discount is 16€ / member / month). However, only active users get billed for each month (but at least 2 users, even if they were active or not). All the users can be Community user. They don´t need additionally an upgrade.
I hope I could help. 
Best regards, 
Zsanett
